README.md

```diff
@@ -79,104 +86,111 @@ Both CLI and programmatic runs support following options:
 | rerunOf   | UUID of launch you want to rerun. If not specified, report portal will update the latest launch with the same name.                                                                                     |
 | debug     | Determines whether newman's run should be logged in details.                                                      |
 
-## Docker
+### Report static attributes
+* To report attributes for suite you should use collection variables.
 
-This reporter can also be used inside of a docker container.
+VARIABLE | INITIAL VALUE | CURRENT VALUE
+--------- | ----------- | -----------
+rp.attributes | keySuiteOne:valueSuiteOne | keySuiteOne:valueSuiteOne
 
-### Using existing image
+* To report attributes for tests inside of Pre-request Script you should use the next method
 
-The docker image for this reporter is available for download from our docker hub. So, first of all you have to ensure that you have docker installed and running in your system. Otherwise, see <a href="https://docs.docker.com/installation/" target="_blank">installation guideline for
-you operating systems</a>.
-
-#### Step 1
+**pm.environment.set**
 
-Pull the <a href="https://hub.docker.com/r/reportportal/newman" target="_blank">newman docker image with installed reporter</a> from docker hub:
+Parameter | Required | Description | Examples
+--------- | ----------- | ----------- | -----------
+namespace | true | "string" - namespace, must be equal to the *rp.attributes* | "rp.attributes"
+attributes | true | "string" - contains set of pairs *key:value* | "keyOne:valueOne;valueTwo;keyThree:valueThree"
 
-```console
-$ docker pull reportportal/newman
+```javascript
+pm.environment.set("rp.attributes", "keyOne:valueOne;valueTwo;keyThree:valueThree");
 ```
+* Step doesn't support reporting with attributes
 
-#### Step 2
+To integrate with Sauce Labs just add attributes:
```

Comment:
Sauce Labs integration available only for items with level `STEP`.
Will it be work?
![image](https://user-images.githubusercontent.com/31861805/110088181-92266e00-7da5-11eb-9213-fecaf3364f28.png)